.github/actions/integration-build/action.yaml

```diff
@@ -0,0 +1,99 @@
+---
+name: "Integration Build"
+description: "Builds and pushes integration images"
+inputs:
+  token:
+    description: "Secret GitHub token"
+    required: true
+  python-version:
+    description: "Python version"
+    required: true
+  branch:
+    description: "Branch name"
+    required: true
+  latest:
+    description: "Tag as latest"
+    required: true
+  build-final:
+    description: "Build final image"
+    required: true
+runs:
+  using: "composite"
```

Comment:
What is `using: "composite"`? I'm used to seeing a Ubuntu version.

Reply:
Composite actions contain multiple steps: https://docs.github.com/en/actions/creating-actions/about-custom-actions#composite-actions

The container used is specified in the calling workflow.